torch/nn/modules/flatten.py

```diff
@@ -9,6 +9,8 @@
 class Flatten(Module):
     r"""
     Flattens a contiguous range of dims into a tensor. For use with :class:`~nn.Sequential`.
+    
+    Inherits from torch.flatten, see https://pytorch.org/docs/stable/generated/torch.flatten.html
```

Comment:
You should be able to create links in the docs like below

```suggestion
    See :meth:`torch.flatten` for details.
```

If you would like you can also check the rendering of the docs under "Preview Python docs built from this PR in the pytorch-bot comment https://github.com/pytorch/pytorch/pull/97276#issuecomment-1478485983

Reply:
ah yes of course!